### ISSUE TITLE
Making it a tutorial?

### ISSUE THREAD

**Issue by omar-3** [rxi/fe#4]
I went through all of your repositories after searching for how to get OOP in ```lua`` and found your very well-written [repository](https://github.com/rxi/classic/) on that topic. I really like your over all projects style and your choices.

I recently came across this [tool](https://github.com/snaptoken/leg) for making tutorial like [that](https://viewsourcecode.org/snaptoken/kilo/index.html), and I just finished studying this tutorial and I learnt ALOT, and was thinking to myself it would be very good if I could do something like that, and then I went through this repo.

I would like to make a tutorial for this repository, what do you think?

**Comment by rxi** [comment]
Thanks for the kind words! Feel free, though it's likely not something I will integrate into the repository itself. Let me know if you do, I would be curious to see the result.

**Comment by omar-3** [comment]
Yes for sure that's is not something to be integrated into the main repository. Thank you and I will post a link here for the tutorial when I get it done.
